packages/cli/package.json

```diff
@@ -161,7 +161,8 @@
     "mock-fs": "^5.1.0",
     "msw": "^1.0.0",
     "nodemon": "^2.0.2",
-    "ts-node": "^10.0.0"
+    "ts-node": "^10.0.0",
+    "type-fest": "^2.19.0"
```

Comment:
You don't actually use this right?

Reply:
A dependency does, and lib check fails without this here